php_zookeeper.c

```diff
@@ -430,6 +485,9 @@ static PHP_METHOD(Zookeeper, set)
 	ZK_METHOD_FETCH_OBJECT;
 
 	if (stat_info) {
+#ifdef ZEND_ENGINE_3
+		ZVAL_DEREF(stat_info);
+#endif
```

Comment:
same block needs instead this line https://github.com/andreiz/php-zookeeper/pull/68/files#diff-4dcc63437fb368928b930d66b19f7b35R414 too, because `get` don't return stat_info by ref